source/libsass.html.haml

```diff
@@ -46,7 +46,9 @@ title: libSass
   %li#java
     :markdown
       ### Java
-      There is one Java wrapper – the [libSass Maven plugin](https://github.com/warmuuh/libsass-maven-plugin).
+      There are (at least) two Java wrappers:
+	  * the [libSass Maven plugin](https://github.com/warmuuh/libsass-maven-plugin);
+	  * the [jsass library](https://github.com/bit3/jsass).
```

Comment:
It looks like your indention is not quite right, here. The bullet points need to be indented one more level in to be read by the markdown filter.